### PR TITLE
chore: move gainsight_px from myaxios to httpget,put and post

### PR DESCRIPTION
**Pull request by ItsSudip**
## What are the changes introduced in this PR?

Updating HTTP module from myaxios to http.

## What is the related Linear task?

Resolves INT-1905

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added in **new readability format**?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3464/test-report.html

File: src/v0/destinations/gainsight_px/util.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 const { NetworkError } = require('@rudderstack/integrations-lib');
-const myAxios = require('../../../util/myAxios');
 const { ENDPOINTS } = require('./config');
 const tags = require('../../util/tags');
 const { getDynamicErrorType } = require('../../../adapters/utils/networkUtils');
 const { JSON_MIME_TYPE } = require('../../util/constant');
+const { httpGET, httpPOST, httpPUT } = require('../../../adapters/network');
```
Comment (sanpj2292) [inline src/v0/destinations/gainsight_px/util.js:6]: Let's use `handleHttpRequest` utility method instead of `httpGET`, `httpPOST` and `httpPUT` ?

Comment (ItsSudip) [reply src/v0/destinations/gainsight_px/util.js]: Addressed

**Review by sanpj2292 — APPROVED** [review]

**Comment by devops-github-rudderstack** [comment]
This PR is considered to be stale. It has been open for 20 days with no further activity thus it is going to be closed in 7 days. To avoid such a case please consider removing the stale label manually or add a comment to the PR.

**Review by yashasvibajpai — APPROVED** [review]

**Pull request merged**